task_11/src/css/urgentSection.scss

```diff
@@ -0,0 +1,149 @@
+@import 'variables.scss';
+
+.urgentSection {
+    width: 50%;
+    margin-top: 62px;
+    margin-left: 4px;
+
+    vertical-align: top;
+    .urgentHeader {
+        padding-left: 72px;
+
+        &:before {
+            left: 2px;
+        }
+    }
+
+    input[type=range] {
```

Comment:
In production it better to use .class selectors, it is more faster. The selectors like this is the slowest.

Reply:
corrected

Reply:
Oh, man, it does not matter. The problem related to big projects and came from 2010 when IE8 was everywhere and computers was not so powerful.

We need to write idiomatic code instead of using unnecessary hacks. Selector __is__ good.

Reply:
Ok, I understood you. I just thought that this is a good practice so far, in case if we need to add another input type range to this page and stylize it in another way, or take this block and move it to other page, if we do a site with a few pages for example.

Reply:
@iammiro you are right too. I did not read the code below, just replied to the comment.

This kind of selector is good for general purposes, but is not good for specific ones.

There is specific, so it's better to use class. But generic one should be present somewhere too.